README.md

```diff
@@ -166,6 +166,11 @@ This summarizes the installation process, the full installation document can be
 
 - Accelerator Prescriptive Architecture Sample [diagrams](./docs/architectures/AWS_Diagrams_Account_Network_VPC.md)
 
+### - Workshops
+
+- **The ASEA Administrator Immersion Day** workshop is focused on helping administrators who will be administering the landing zone understand how they can design, build and operate the components in ASEA. Click [here](https://catalog.us-east-1.prod.workshops.aws/v2/workshops/f3ed5d0f-d2f1-47e8-a305-168da9179aaa/en-US/sea-administrators) for an overview of the topics covered.
+- **The ASEA Project Team Immersion Day** workshop is focused on helping project teams understand what it means to operate within an ASEA managed environment. Click [here](https://catalog.us-east-1.prod.workshops.aws/v2/workshops/f3ed5d0f-d2f1-47e8-a305-168da9179aaa/en-US/sea-members) for an overview of the topics covered.
```

Comment:
- change ASEA to Accelerator per above
- would "Workload/Application Team" be clearer instead of "Project"?

Reply:
- I updated from ASEA to Accelerator
- Also changed it to Workload/Application team for this context (remains Project on the workshop site)